core/src/main/java/org/eclipse/hono/util/ResourceIdentifier.java

```diff
@@ -286,6 +294,9 @@ public String getBasePath() {
 
     /**
      * Gets a string representation of the resource identifiers' parts without the base path.
+     * <p>
+     * E.g. for a resource <em>control/myTenant/deviceId/some/path</em>, the return value will be
+     * <em>deviceId/some/path</em>.
```

Comment:
we should also document that this method returns an empty string (instead of `null`), if the identifier doesn't contain any additional path segments.

Reply:
done